src/algebra/category/Module/simple.lean

```diff
@@ -27,3 +28,22 @@ simple_iff_is_simple_module.mpr ‹_›
 /-- A simple object in the category of modules is a simple module. -/
 instance is_simple_module_of_simple (M : Module R) [simple M] : is_simple_module R M :=
 simple_iff_is_simple_module.mp (simple.of_iso (of_self_iso M))
+
+open finite_dimensional
+
+/-- Any `k`-algebra module which is 1-dimensional over `k` is simple. -/
+lemma simple_of_finrank_eq_one {k : Type*} [field k] [algebra k R]
+  {V : Module R} (h : finrank k V = 1) : simple V :=
```

Comment:
Can you prove this for `is_simple_module` too?

Reply:
There's an `iff` directly above that handles this. I'm not sure I want to translate every single statement going forward...

Reply:
Right, but doesn't the `is_simple_module` version work when `R` and `V` are in different universes? I thought the approach was usually to prove the universe-polymorphic version outside of category theory, then give the short translation from that into the category theory land.

Reply:
Actually, I think the category theory land statement is (or could be?) fully universe polymorphic in this case, so that, by itself, isn't reason enough to duplicate. I agree, though, that this requires more thought than I gave it initially.

A related question which will come up soon: we already have two versions of a baby-Schur's lemma (endomorphisms of a simple form a division ring), for an arbitrary preadditive category, and for a R-modules. However we currently only have proper Schur's lemma (endomorphisms of a simple, in a k-linear category, where k is algebraically closed, are scalar multiples of the identity) in the general category case. Clearly (!) in this instance we should be ripping out the R-module proof and deducing it from the general statement, right?

(Note that this question is completely different than the one at hand in this PR, because today's fact is really about R-modules.)

Reply:
Okay, @eric-wieser, I have "decategorified" the proof now.

Reply:
Thanks!